src/main/java/net/dv8tion/jda/api/entities/Role.java

```diff
@@ -448,4 +465,126 @@ default String getSubscriptionId()
          */
         boolean isLinkedRole();
     }
+
+    /**
+     * Colors associated with this role.
+     */
+    interface RoleColors
+    {
+        /**
+         * A value representing not set color.
+         */
+        int COLOR_NOT_SET = -1;
+
+        /**
+         * The raw color RGB value used for this role.
+         * <p>Defaults to {@link #DEFAULT_COLOR_RAW} if this role has no set color.
+         * <p>In case this role's coloring style is {@link Style#HOLOGRAPHIC HOLOGRAPHIC}, this method will return {@code 11127295}.</p>
```

Comment:
Avoid closing paragraphs in docs:


```suggestion
         * <p>In case this role's coloring style is {@link Style#HOLOGRAPHIC HOLOGRAPHIC}, this method will return {@code 11127295}.
```